internal/capability/auditor.go

```diff
@@ -10,30 +10,29 @@ import (
 // It has methods to create a workqueue with all the package and audit requirements for a
 // particular audit run
 type Auditor interface {
```

Comment:
I guess to keep it in a minimal level it would be only CapAudit exported. But yeah future implementation may require an interface if we want to imagine different auditor implementations. But I removed it anyway. If we need in the future we bring it back. There we go 18d4706